FlooringOrderingSystem/FlooringOrderingSystem.Data/ReferenceDataRepository/ProductsRepository.cs

```diff
@@ -0,0 +1,131 @@
+﻿using FlooringOrderingSystem.Models;
+using FlooringOrderingSystem.Models.Responses;
+using FlooringOrderingSystem.Models.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FlooringOrderingSystem.Data.ReferenceDataRepository
+{
+    public class ProductsRepository:IProductsRepository
+    {
+        private readonly string path = @"C:\Users\Rohit\Desktop\SoftwareGuild\Repos\FlooringOrderingSystem\FlooringOrderingSystem.Data\DataFiles\Products.txt";
```

Comment:
Same as above, do not use hard coded file paths

Reply:
Done. But as I said above, doing this is failing Nunit Test since Nunit is running from a different level than the application